modules/ROOT/pages/value-providers.adoc

```diff
@@ -74,7 +74,8 @@ public class StaticUserRoleValueProvider implements ValueProvider {
 +
 . Mark the parameter that requires dynamic values with the `@OfValues()` annotation.
 +
-`@OfValues` requires a `ValueProvider` implementation that is bound to the parameter, for example:
+`@OfValues` requires a `ValueProvider` implementation that is bound to the parameter.
+For example, the following operation method `publishAs` includes binding:
```

Comment:
Maybe it sounds better to say `includes such binding` ?

Reply:
yes, thanks!